source/services/BookablesService.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from "../helpers/ApiRequest";
+
+export type BookableItem = {
+  name: string;
+  sharedMailbox: string;
+  address: string;
+  formId: string;
+};
+
+async function getBookables(): Promise<BookableItem[]> {
```

Comment:
You are using async keyword but not async/await syntax. Perhaps a good idea to change it